core/commands/partials/vagrant_configuration_generator.rb

```diff
@@ -349,15 +275,10 @@ def generate_node_defenition(node_params, cookbook_path, path)
     case node_params[:provider]
     when 'virtualbox'
       get_virtualbox_definition(cookbook_path, node_params)
-    when 'aws'
-      tags = generate_aws_tag('hostname' => Socket.gethostname, 'username' => Etc.getlogin,
-                              'full_config_path' => File.expand_path(path), 'machinename' => node_params[:name])
-      node_params[:device_name] = @aws_service.device_name_for_ami(node_params[:ami])
```

Comment:
The corresponding method of AWS Service is no longer in use. It is strange, we should have been enlarging the base for AWS images. Does this functionality is still present?